src/App.vue

```diff
@@ -151,11 +153,23 @@ export default {
         debouncedEmitUserSessionExpiredError()
       })
     },
+    isAllowedOrigin (origin) {
```

Comment:
This is a method rather than a computed property.

Reply:
Yes thanks.

Reply:
Hmm, I checked it is a method.
Github display misleads us.